test/models/skill_inventory_test.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../test_helper'
+
+class SkillInventoryTest < Minitest::Test
+  include TestHelpers
+  attr_reader :skill, :id
+
+  def setup
```

Comment:
I like that you are trying to use #setup, but if you don't need everything you have in the setup for every test, you shouldn't user setup.